endpoints.json

```diff
@@ -156,5 +99,80 @@
         }
       ]
     }
+  },
+  "GET /api/articles?topic": {
+    "description": "serves a filtered array of articles where the article topic matches the topic query",
+    "example query": "/api/articles?topic=cooking",
+    "exampleResponse": {
+      "articles": [
+        {
+          "title": "Seafood substitutions are increasing",
+          "topic": "cooking",
+          "author": "weegembump",
+          "created_at": "2018-05-30T15:59:13.341Z",
+          "votes": 0,
+          "comment_count": "6",
+          "article_id": 2,
+          "article_img_url": "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
+        }
+      ]
+    }
+  },
```

Comment:
This shouldnt be added as a new entry, but added on to the existing base endpoint, but with the queries array added to look like the following:
`"queries": ["topic"]`